docs/cluster_up_down.md

```diff
@@ -229,14 +243,18 @@ To stop the cluster, run:
 C:\> oc cluster down
 ```
 
-### Windows with Docker Toolbox
+### Windows with Docker Machine
 
-1. Install [Docker Toolbox](https://www.docker.com/products/docker-toolbox) and ensure that it is functional.
-2. Download the Windows `oc.exe` binary from [openshift-origin-client-tools-VERSION-windows.zip](https://github.com/openshift/origin/releases) and place it in your path.
+OpenShift is currently compatible with version 17.09.1 of Docker for Windows.
+
+1. Install [Docker for Windows](https://docs.docker.com/docker-for-windows/release-notes/) making sure you meet the [prerequisites](https://docs.docker.com/docker-for-windows/#/what-to-know-before-you-install)
+and download a compatible version of Docker.
+2. Review the [prerequisite instructions](https://docs.docker.com/machine/get-started/#prerequisite-information) to set up Docker Machine on Windows.
+3. Download the Windows `oc.exe` binary from [openshift-origin-client-tools-VERSION-windows.zip](https://github.com/openshift/origin/releases) and place it in your path.
 
    > Please be aware that the 'oc cluster' set of commands are only available in the 1.3+ or newer releases.
 
-3. Open a Command window as Administrator (for most drivers, docker-machine on Windows requires administrator privileges)
+4. Open a Command window as Administrator (for most drivers, docker-machine on Windows requires administrator privileges)
```

Comment:
same here, --create-machine is getting removed. Point to external doc on how to create the machine and set up the Docker environment before running `oc cluster up`

Reply:
Since `--version` is being replaced with `--tag`, should we instruct _contributing_ developers to build the client from source via `hack/build-go.sh cmd/oc`?

Reply:
yes we'd expect contributing developers to build oc from source, but this doc(cluster_up_down) isn't for contributing developers, so i'm not sure where you're proposing making that suggestion.